### PR TITLE
refactor: remove interfaces (ffi, wasm and pyo3)

### PR DESCRIPTION
**Pull request by Manuthor**

### CODE REVIEW

File: src/test_utils/tests_data/legacy_policy.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"last_attribute_value":7,"max_attribute_creations":100,"axes":{"Department":[["R&D","HR","MKG","FIN"],false],"Security Level":[["Protected","Confidential","Top Secret"],true]},"attributes":{"Department::MKG":[6],"Department::FIN":[7],"Department::HR":[5],"Security Level::Top Secret":[3],"Security Level::Protected":[1],"Security Level::Confidential":[2],"Department::R&D":[4]}}
```
Comment (tbrezot) [inline src/test_utils/tests_data/legacy_policy.json:1]: Github does not signal the absence of new line character at the end of this file: is it normal?

**Review by tbrezot — CHANGES_REQUESTED** [review]

File: src/test_utils/non_regression.rs [hunk 2]
```diff
@@ -38,19 +39,21 @@ pub fn policy() -> Result<Policy, Error> {
 //#[test]
 //fn write_policy() {
 //let _policy = policy().unwrap();
-//std::fs::write("tests/policy.json", serde_json::to_vec(&_policy).unwrap()).unwrap();
+//std::fs::write("tests/policy.json",
+// serde_json::to_vec(&_policy).unwrap()).unwrap();
```
[the remark this answers is not in the document]
Comment (Manuthor) [reply src/test_utils/non_regression.rs]: done

File: src/test_utils/non_regression.rs [hunk 3]
```diff
@@ -255,6 +259,7 @@ impl NonRegressionTestVector {
         Ok(reg_vectors)
     }
 
+    #[allow(dead_code)]
```
[the remark this answers is not in the document]
Comment (Manuthor) [reply src/test_utils/non_regression.rs]: thanks

File: examples/decrypt.rs [hunk 4]
```diff
@@ -1,16 +1,18 @@
+use cosmian_cover_crypt::statics::{CoverCryptX25519Aes256, EncryptedHeader, UserSecretKey};
+
 fn main() {
-    #[cfg(feature = "interfaces")]
+    #[cfg(feature = "interface")]
     {
-        use cosmian_cover_crypt::{
-            api::EncryptedHeader, interfaces::statics::UserSecretKey, CoverCryptStruct,
-        };
+        // use cosmian_cover_crypt::{
```
[the remark this answers is not in the document]
Comment (Manuthor) [reply examples/decrypt.rs:6]: done

File: src/test_utils/non_regression.rs [hunk 5]
```diff
@@ -66,6 +69,7 @@ struct EncryptionTestVector {
 }
 
 impl EncryptionTestVector {
+    #[allow(dead_code)]
```
[the remark this answers is not in the document]
Comment (Manuthor) [reply src/test_utils/non_regression.rs]: thanks

File: Cargo.toml [hunk 6]
```diff
@@ -25,14 +25,12 @@ name = "benches"
...
 [features]
-ffi = ["cosmian_ffi", "lazy_static", "interface"]
 full_bench = ["interface"]
 hybridized_bench = []
-python = ["pyo3", "interface"]
-wasm_bindgen = ["js-sys", "wasm-bindgen", "interface"]
 interface = []
 
 [dependencies]
+base64 = "0.13"
```
[the remark this answers is not in the document]
Comment (Manuthor) [reply Cargo.toml:33]: it is due to the new module `test_utils` where base64 is required

[the remark this answers is not in the document]
Comment (tbrezot) [reply Cargo.toml:33]: It seems it is just in the `non_regression` module which is optional. Please, make this dependency optional too.

[the remark this answers is not in the document]
Comment (Manuthor) [reply Cargo.toml]: right

**Pull request merged**